brightchain-lib/src/lib/serializableBuffer.ts

```
@@ -1,14 +1,688 @@
+  // Buffer pooling methods
+  private static getFromPool(size: number): Buffer | undefined {
+    const pool = this.bufferPool.get(size);
+    return pool?.pop();
+  }
+
+  private static returnToPool(buffer: Buffer): void {
+    const size = buffer.length;
+    if (!this.bufferPool.has(size)) {
+      this.bufferPool.set(size, []);
+    }
+    const pool = this.bufferPool.get(size)!;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)